### ISSUE TITLE
Adding a direct npm-package to repository-url endpoint

### ISSUE THREAD

**Issue by caub** [npms-io/npms-api#75]
It's very useful as a browser search engine

Currently I've added [https://caub.github.io/misc/npm-to-gh?q=%s](https://caub.github.io/misc/npm-to-gh) in my browser search engines, as 'npm', then I can quickly type `npm` `TAB` `some-package` `ENTER`

source: https://github.com/caub/misc/blob/master/npm-to-gh.html#L32-L42

if you could handle it directly it would be ideal, and remove the need of that hacky script

**Comment by satazor** [comment]
@caub I'm not sure what you meant with this issue. Both the search and package info endpoints contain the repository url, e.g.: https://api.npms.io/v2/search?from=0&q=foo&size=1 and https://api.npms.io/v2/package/foo.

Can you elaborate?

**Comment by caub** [comment]
@satazor yes that's what I'm using https://github.com/caub/misc/blob/master/npm-to-gh.html#L51

What I mean is having an endpoint like `https://api.npms.io/v2/package/foo/url?redirect` that'd reply with a `Location: https://github.com/jasonLaster/foo` header and 307 status
